test/scripts/deploy_with_helm.sh

```diff
@@ -89,7 +89,7 @@ fi
 
 AUTH_ARGS=""
 if [ "$AUTH" ]; then
-  AUTH_ARGS="--set global.auth.type=builtin"
+  AUTH_ARGS="--set global.auth.type=oidc"
```

Comment:
Is this a leftover for testing?
With the standalone deployment, the `keycloak-db` SS is not created and `deploy_with_helm.sh` fails when it checks for it a bit later

 `kubectl rollout status statefulset keycloak-db -n flightctl-external -w --timeout=300s --context kind-kind`

Reply:
Yes, it is! Fixed